arch/inst/F/fadd.s.yaml

```diff
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=../../../schemas/inst_schema.json
+
+fadd.s:
+  long_name: No synopsis available.
+  description: |
+      No description available.
+  definedBy: F
+  assembly: xd, xs1, xs2, rm
+  encoding:
+    match: 0000000------------------1010011
+    variables:
+    - name: rs2
```

Comment:
is it possible to get the names of the variables as "fs2" when they are floating point registers? This would also apply to the assembly string. I know this isn't quite how the ISA manuals treat things, but having the register file explicit helps with machine parsing.

Reply:
Surely, I think this will need to be manually but I'll get to it and fix them.

Reply:
@dhower-qc do you mind taking them on this not fully correct format for now? Just to kind of get all instruction skeletons in place and once that's done come back to this known issues and fix the one by one? I understand that we'll probably be creating some tech debt with this, but given the proximity to the summit, I think it could be beneficial to have all the skeletons in. BTW I'll gladly do whatever option you prefer, I just don't think we'll be able to get all skeletons by the 22nd.

Reply:
Done.